src/Cardano/BM/Configuration/Model.lhs

```diff
@@ -433,7 +497,25 @@ setupFromRepresentation r = do
 \begin{code}
 empty :: IO Configuration
 empty = do
-    cgref <- newMVar $ ConfigurationInternal Debug HM.empty HM.empty HM.empty HM.empty [] [] HM.empty HM.empty [] [] HM.empty StatsAK 0 0
```

Comment:
this is not very informative!
I have a change in the pipeline where we set the fields explicitly. Just making sure it does not get overwritten.

Reply:
https://github.com/input-output-hk/iohk-monitoring-framework/pull/122/commits/d62a01966c2eda2de5761e2aa8a15ca1a875b4a1